robot/basestation/app.py

```diff
@@ -22,20 +22,20 @@
 
 app = flask.Flask(__name__)
 
-def run_shell(cmd, args=""):
+def run_shell(cmd, args="", print_output=True):
```

Comment:
Did you ever use the parameter you added?

Reply:
I did, I used it in the branch name verification hook. In order to stop run_shell from printing stuff in the terminal after every checkout.